demos-europe/demosplan-core
# feat(refs DPLAN-11448): Fix split statement

**Ticket:** DPLAN-11448

- Add required prop statementId to addon props
- Remove unnecessary store action (action is in addon)

### How to review/test
- Install Addon `bin/<project> dp:addon:install --github --branch` 
- Verfahren -> Stellungnahmen -> Aufteilen -> You should be able to split statement
